composer.json

```diff
@@ -21,24 +21,24 @@
         "doctrine/annotations": "^1.2",
         "gecko-packages/gecko-php-unit": "^2.0",
         "sebastian/diff": "^1.4",
-        "symfony/console": "^2.4 || ^3.0",
-        "symfony/event-dispatcher": "^2.1 || ^3.0",
-        "symfony/filesystem": "^2.4 || ^3.0",
-        "symfony/finder": "^2.2 || ^3.0",
-        "symfony/options-resolver": "^2.6 || ^3.0",
+        "symfony/console": "^2.4 || ^3.0 || ^4.0",
+        "symfony/event-dispatcher": "^2.1 || ^3.0 || ^4.0",
+        "symfony/filesystem": "^2.4 || ^3.0 || ^4.0",
+        "symfony/finder": "^2.2 || ^3.0 || ^4.0",
+        "symfony/options-resolver": "^2.6 || ^3.0 || ^4.0",
         "symfony/polyfill-php54": "^1.0",
         "symfony/polyfill-php55": "^1.3",
         "symfony/polyfill-php70": "^1.0",
         "symfony/polyfill-php72": "^1.4",
-        "symfony/process": "^2.3 || ^3.0",
-        "symfony/stopwatch": "^2.5 || ^3.0"
+        "symfony/process": "^2.3 || ^3.0 || ^4.0",
+        "symfony/stopwatch": "^2.5 || ^3.0 || ^4.0"
     },
     "require-dev": {
         "johnkary/phpunit-speedtrap": "^1.0.1",
         "justinrainbow/json-schema": "^5.0",
         "phpunit/phpunit": "^4.8.35 || ^5.4.3",
-        "php-coveralls/php-coveralls": "^1.0.2",
-        "symfony/phpunit-bridge": "^3.2.2"
+        "satooshi/php-coveralls": "^1.0.2@dev",
```

Comment:
please drop `@dev`. 1.0.2 is `@stable`.

Reply:
even more, sth wrong with your rebase.
it shall be `"php-coveralls/php-coveralls"`, not `"satooshi/php-coveralls"`